test/__snapshots__/index.test.js.snap

```diff
@@ -1176,7 +1177,7 @@ exports[`fixtures build css-modules--string with microbundle 3`] = `
 `;
 
 exports[`fixtures build css-modules--string with microbundle 4`] = `
-"export default function(){var a=document.createElement(\\"div\\");return a.className=\\"_contains_this_0a8c24df242c2cd708036873307aea94 _contains_this_81567d0efc15a456670452d3277e1a68\\",a}
+"var b={test_class_that_should_be_scoped:\\"_contains_this_81567d0efc15a456670452d3277e1a68\\"},c={scoped_class:\\"_contains_this_0a8c24df242c2cd708036873307aea94\\"};export default function(){var a=document.createElement(\\"div\\");a.className=c.scoped_class+\\" \\"+b.test_class_that_should_be_scoped;return a}
```

Comment:
these are a bit disappointing I guess

Reply:
It's an object coming from an import - I'm thinking closure can't inline because its crossing a module boundary?

@kristoferbaxter maybe cross-module code motion is only enabled for ADVANCED_OPTIMIZATIONS?

Reply:
Correct @developit